data/gauge_01073000/example_bmi_multi_realization_config_w_routing_w_coastal.json

```diff
@@ -0,0 +1,124 @@
+                            "name": "bmi_c",
+                            "params": {
+                                "model_type_name": "bmi_c_cfe",
+                                "library_file": "./extern/cfe/cmake_build/libcfebmi.so",
+                                "forcing_file": "",
+                                "init_config": "./data/gauge_01073000/CFE/{{id}}_bmi_config_cfe_pass.txt",
+                                "allow_exceed_end_time": true,
+                                "main_output_variable": "Q_OUT",
+                                "registration_function": "register_bmi_cfe",
+                                "variables_names_map": {
+                                    "atmosphere_water__liquid_equivalent_precipitation_rate": "QINSUR",
+                                    "water_potential_evaporation_flux": "water_potential_evaporation_flux",
+                                    "ice_fraction_schaake" : "sloth_ice_fraction_schaake",
+                                    "ice_fraction_xinanjiang" : "sloth_ice_fraction_xinanjiang",
+                                    "soil_moisture_profile" : "sloth_smp"
+                                },
+                                "uses_forcing_file": false
+                            }
+                        }
+                    ],
+                    "uses_forcing_file": false
+                }
+            }
+        ],
+        "forcing": {
+            "file_pattern": ".*{{id}}.*.csv", 
+            "path": "./data/gauge_01073000/forcing",
+            "provider": "CsvPerFeature"
+        }
+    },
+    "time": {
+        "start_time": "2024-02-20 12:00:00",
+        "end_time": "2024-02-20 23:00:00",
+        "output_interval": 3600
+    },
+    "routing": {
+        "t_route_config_file_with_path": "./data/gauge_01073000/routing_config.yaml"
+    },
+    "coastal": {
+                 "name": "bmi_fortran",
+                 "params": {
+                                "model_type_name": "schism_coastal_formulation",
+                                "library_file": "/contrib/Zhengtao.Cui/home/ngwpc/schism/build/lib/libschism_bmi.so",
+                                "met_forcing_netcdf_path": "/efs/coastal_testdata/hrrr_scratch/NextGen_Forcings_Engine_MESH_output_202402201200.nc",
+                                "offshore_boundary_netcdf_path": "/mnt/SCHISM_Lake_Champlain_BMI_test/NextGen_Forcings_Engine_MESH_offshsort_output_201104302300.nc",
+                                "streamflow_boundary_netcdf_path": "/contrib/Zhengtao.Cui/home/ngwpc/ngen_coastal/data/SCHISM_Hawaii/source_TRoute.nc",
+				"working_dir" : "/contrib/Zhengtao.Cui/home/ngwpc/ngen_coastal/data/SCHISM_Lake_Champlain_BMI_Driver_Test/",
+				"model_start_time_in_secs" : "0",
+				"nscribs" : "0",
+                                "allow_exceed_end_time": true,
+                                "main_output_variable": "z",
+                                "uses_forcing_file": false
```

Comment:
These are legacies of the catchment formulations, with no applicability to the coastal model, right? Let's just drop them.